src/components/Header/index.js

```diff
@@ -1,18 +1,42 @@
 const React = require('react');
+const T = require('prop-types');
 const NavLink = require('react-router-dom').NavLink;
 const Classes = require('./styles.scss');
 
-module.exports = () => (
-
-    <div>
-        <h1>Strangeluv</h1>
-        <NavLink exact to='/' activeClassName={Classes.activeRoute}>
-            Home
-        </NavLink>
-        {' · '}
-        <NavLink to='/counter' activeClassName={Classes.activeRoute}>
-            Counter
-        </NavLink>
-    </div>
-
-);
+module.exports = class Header extends React.Component {
+
+    static propTypes = {
+        logout: T.func.isRequired,
+        isAuthenticated: T.bool.isRequired
+    };
+
+    render() {
+
+        const { isAuthenticated, logout } = this.props;
+
+        return (
+
+            <div>
+                <h1>Strangeluv</h1>
+                <NavLink exact to='/' activeClassName={Classes.activeRoute}>
+                    Home
+                </NavLink>
+                {' · '}
+                <NavLink to='/dashboard' activeClassName={Classes.activeRoute}>
+                    Dashboard
+                </NavLink>
+                {' · '}
+                {!isAuthenticated &&
+                    <NavLink to='/sign-up' activeClassName={Classes.activeRoute}>
+                        Sign Up
+                    </NavLink>
+                }
+                {!isAuthenticated &&
```

Comment:
You might be able to use a [fragment](https://reactjs.org/docs/fragments.html) to avoid `!isAuthenticated` twice in a row.